pkg/components/standalone_loader_test.go

```diff
@@ -1,72 +1,13 @@
 package components
 
 import (
-	"fmt"
-	"io/ioutil"
 	"testing"
 
-	"bou.ke/monkey"
 	"github.com/stretchr/testify/assert"
 
 	config "github.com/dapr/dapr/pkg/config/modes"
 )
 
-func TestLoadComponentsFromFile(t *testing.T) {
```

Comment:
Just a thought, rather than remove the test - would it not be fairly simple to inject an `io.Reader` and change `loadComponentsFromFile` to use that or even just write to a temporary file to keep the test but remove the `bou.ke/monkey` dependency?

Reply:
P.S. Didn't realize @yaron2 had already comments something to this effect.